sklearn/compose/_column_transformer.py

```diff
@@ -471,7 +471,23 @@ def transform(self, X):
             # All transformers are None
             return np.zeros((X.shape[0], 0))
 
-        return _hstack(list(Xs), self.sparse_output_)
+        return self._hstack(list(Xs), self.sparse_output_)
+
+    def _hstack(self, X, sparse_):
```

Comment:
Now that this is a method you can directly access `self.sparse_output_` instead of passing `sparse_` as an argument.